src/Command/WorkerCommand.php

```diff
@@ -48,8 +48,7 @@ public function getOptionParser(): ConsoleOptionParser
             'short' => 'c',
         ]);
         $parser->addOption('queue', [
-            'default' => 'default',
-            'help' => 'Name of queue to bind to',
+            'help' => 'Name of queue to bind to. Defaults to the queue config (--config).',
```

Comment:
Should this use enqueue terms such as `topic` instead?

Reply:
I don't know. We don't use topic anywhere else and switching would require updating a few different places.

Reply:
I was thinking just for documentation since this is directly tied to bindTopic.